src/doc/src/reference/workspaces.md

```diff
@@ -6,36 +6,91 @@ various settings such as profiles. Packages that are part of a workspaces are
 called *workspace members*. There are two flavours of workspaces: as root
 package or as virtual manifest.
 
-### Root package
+The key points of workspaces are:
+
+* All packages share a common `Cargo.lock` file which resides in the
+  *workspace root*.
+* All packages share a common [output directory], which defaults to a
+  directory named `target` in the *workspace root*.
+* The [`[patch]`][patch], [`[replace]`][replace] and [`[profile.*]`][profiles]
+  sections in `Cargo.toml` are only recognized in the *root* manifest, and
+  ignored in member crates' manifests.
+
+<a id="package-selection"></a>
+In a workspace, package-related cargo commands like [`cargo build`] can use
+the `-p` / `--package` or `--workspace` command-line flags to determine which
+packages to operate on. If neither of those flags are specified, Cargo will
+use the package in the current working directory. If the current directory is
+a virtual workspace, it will apply to all members (as if `--workspace` were
```

Comment:
(nit): This should probably link to `virtual workspace` since it has not been introduced before being mentioned here